src/lib/sql/sql_translator.cpp

```diff
@@ -475,7 +475,7 @@ SQLTranslator::TableSourceState SQLTranslator::_translate_table_origin(const hsq
         }
 
         AssertInput(_use_mvcc == (lqp_is_validated(view->lqp) ? UseMvcc::Yes : UseMvcc::No),
```

Comment:
@mrks Do we even want to run the TPCH tests with encodings? This takes a while (mostly due to Q21, 5 Minutes here):
![image](https://user-images.githubusercontent.com/2679018/49003821-0c95cb80-f163-11e8-9464-f4aa25af4730.png)

Maybe running the SQL tests with encoding is sufficient? CI is slow enough as it is, after all.

Reply:
Is this debug or release? Maybe release is sufficient?

Reply:
This is debug. But as you said in #1322 the system tests will run through memcheck, too, which is going to be even slower than debug mode.

Reply:
Let's put this PR on ice and discuss this in person.

Reply:
As (hopefully) discussed: I changed the TPCH Test to only run JIT/non-JIT and NOT with different encodings